test/simulation/README.md

```diff
@@ -15,33 +14,93 @@ Requirements:
 Installation & Usage:
 
 ```bash
-$ export PATH=$PATH:$PWD/go/bin
-$ GOPATH=$PWD/go GO111MODULE=on go test -v
+export PATH=$PATH:$PWD/go/bin
+GOPATH=$PWD/go GO111MODULE=on go test -v
 ````
 
-## Network Scenarios Tests
+## Simulation Test Cases
+
+Below is a list of implemented (checked) and planned (unchecked) test cases.
+
 ### Initialization
+
 - [x] Verify connectivity to swap clients of all nodes.
 - [x] Add currencies and pairs to all nodes.
 
 ### P2P
+
 - [x] Discovery: Connect Alice to Bob, Bob to Carol, Dave to Carol. After a short while, all of them should end up connected.
 - [x] Incorrect public key: Alice connects to Bob expecting an incorrect public key. Bob should reject the connection due to auth invalid target.
 - [x] Banning: If Alice ban Bob, connection attempts from both directions should fail.
 - [x] Unbanning: If Alice unban Bob, connection attempts from both directions should succeed.
 - [x] Duplicate connection: If Alice and Bob are already connected, connection attempts from both directions should fail.
 
 ### Decentralized Order Book
+
 - [x] Placed order should get broadcasted over the network, and added to connected peers' order books.
 - [x] Removed order should get invalidated over the network, and removed from connected peers' order books.
 - [ ] Placed order should get internal matches, and trigger order invalidation over the network.
 - [ ] Peer disconnection should trigger orders removal to all his orders.
 
 ### Node State
+
 - [ ] Removed pair or swap client (lnd/raiden) disconnected (after timeout elapsed) should trigger order removal for all the pair's orders, and removed from connected peers' order books.
 - [ ] Updated `LND-BTC`/`LND-LTC` public keys should propagate over the network.
 
 ### Swaps
+
 - [x] Placed order should trigger a swap.
 - [ ] Placed order should trigger a swap and order invalidation on the maker order to all connected peers besides the taker peer.
 - [x] Swaps that use multihop routes for payment on lnd.
+
+## Adversarial Test Cases
+
+Adversarial test cases pit a modified version of `xud` against a standard one to see how it deals with malicious and/or unexpected behavior from other nodes. These are intended to measure and improve the robustness of `xud` against attacks, faulty nodes or connections, and potential loss of funds.
+
+### Swap partners go offline or become unresponsive
+
+- [x] Taker becomes unresponsive after maker confirms quantity with a `SwapAcceptedPacket`.
+- [ ] Maker becomes unresponsive after maker confirms quantity with a `SwapAcceptedPacket`.
+- [ ] Taker becomes unresponsive after taker set up first HTLC to the maker.
+- [x] Maker becomes unresponsive after taker set up first HTLC to the maker (to be fixed: <https://github.com/ExchangeUnion/xud/issues/1048>).
```

Comment:
We have 2 distinct tests: one when the maker shutdown (`testMakerShutdownAfter1stHTLC`), the other when it just becomes unresponsive to the swap protocol (`testMakerStallingAfter1stHTLC`).
The mentioned issue (https://github.com/ExchangeUnion/xud/issues/1048) is for the first, and not the later. 
Also, I think we agreed that it's not really a problem we should close this issue, no?

Reply:
> Also, I think we agreed that it's not really a problem we should close this issue, no?
Document how to solve it, then can be closed.